src/Altinn.Platform/Altinn.Platform.Events/Events/Authorization/AuthorizationHelper.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Constants;
+using Altinn.Common.PEP.Helpers;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Events.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Platform.Events.Authorization
+{
+    /// <summary>
+    /// Authorization Helper for 
+    /// </summary>
+    public class AuthorizationHelper
+    {
+        private readonly IPDP _pdp;
+ 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AuthorizationHelper"/> class.
+        /// </summary>
+        /// <param name="pdp">The policy decision point</param>
+        /// <param name="logger">The logger to use by the class.</param>
+        public AuthorizationHelper(IPDP pdp)
+        {
+            _pdp = pdp;
+         }
+
+        /// <summary>
+        /// Authorizes and filters events based on authorization
+        /// </summary>
+        /// <param name="consumer">The vent consumer</param>
+        /// <param name="cloudEvents">The list of events</param>
+        /// <returns></returns>
```

Comment:
remove tag or fill in returned description

Reply:
fixed